docs/v3_services.rst

```diff
@@ -0,0 +1,237 @@
+Securedrop V3 Onion Services
+============================
+
+SecureDrop uses onion services for the *Journalist* and *Source Interfaces*, as
+well as for SSH access in SSH-over-Tor mode. Previous to the
+release of SecureDrop 1.0.0, only v2 onion services were available, but 
+administrators now have the option of enabling next-generation (v3) onion 
+services.
+
+V3 onion services use more secure cryptography than v2 onion services,  and 
+include redesigned protocols that guard against service 
+information leaks on the Tor network. For new SecureDrop instances, it
+is recommended to enable v3 onion services only. For existing instances, 
+migration to v3 onion services is recommended, and the ability to run both
+v2 and v3 onion services simultaneously is supported to make the process
+easier.
```

Comment:
You mention the 56 character addresses later; I would recommend putting this early in the introduction, as well, perhaps with an example address for v2 and v3.

Reply:
:heavy_check_mark: